qemu/tests/cfg/blockdev_commit_fio.cfg

```diff
@@ -0,0 +1,22 @@
+- blockdev_commit_fio:
+    type = blockdev_commit_fio
+    virt_test_type = qemu
+    only Linux
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    snapshot_tags = sn1
+
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+
+    device_tag = "image1"
+    rebase_mode = unsafe
+    qemu_force_use_drive_expression = no
+    mount_point = "/tmp"
```

Comment:
@aliang123 dd write with 'oflag=direct' to the 'tmp' directory would fail, shall we change it?

 aexpect.exceptions.ShellCmdError: Shell command failed: 'dd if=/dev/urandom of=/tmp/sn1 bs=1M count=10 oflag=direct'    (status: 1,    output: "dd: failed to open '/tmp/sn1': Invalid argument\n")

Reply:
'/tmp' is a folder may use 'tmpfs' format, which does not same data to the disk, and will be cleaned after reboot('/tmp' is wiped whenever the system reboots where as /var/tmp gets preserved across reboots).
suggestion: save file under '/var/tmp' instead of '/tmp'

Reply:
Have updated it to "var/tmp", thanks @ngu-niny and @XueqiangWei . @ngu-niny, please help to review again, thanks.